templates/docs/examples/patterns/accordion/default.html

```diff
@@ -4,11 +4,11 @@
 {% block standalone_css %}patterns_accordion{% endblock %}
 
 {% block content %}
-<aside class="p-accordion" role="tablist" aria-multiselect="true">
+<aside class="p-accordion" role="tablist" >
   <ul class="p-accordion__list">
     <li class="p-accordion__group">
-      <button type="button" class="p-accordion__tab" id="tab1" role="tab" aria-controls="tab1-section" aria-expanded="true">Advanced topics</button>
-      <section class="p-accordion__panel" id="tab1-section" role="tabpanel" aria-hidden="false" aria-labelledby="tab1-section">
+      <button type="button" class="p-accordion__tab" id="tab1" aria-controls="tab1-section" aria-expanded="true">Advanced topics</button>
+      <section class="p-accordion__panel" id="tab1-section" aria-hidden="false" aria-labelledby="tab1-section">
```

Comment:
I just noticed that is seems that our `aria-labelledby` may be wrong on all accordion examples.
It seems to be pointing to itself, which doesn't make much sense. I think it was supposed to be pointing to the accordion header button (that is a 'label' of the section).

So it would be `aria-labelled-by="tab1"` in this case.

It's the same error in all accordion examples as far as I can see.